features/step_definitions/upgrade/admin_backup_steps.rb

```diff
@@ -0,0 +1,48 @@
+Given(/^I click the "([^"]*)" button on the landing page$/) do |button_text|
+  visit('/upgrade')
+  # We need to wait a bit to get the final page with correct url
+  sleep 10
+  if page.current_url.match(/landing$/)
+    verify_upgrade_landing_page
+    upgrade_button = find_button(button_text)
+    upgrade_button.click
+    expect(upgrade_button.find("suse-lazy-spinner").visible?).to be(true)
+  end
+
+  wait_for "Starting upgrade workflow", max: "80 seconds", sleep: "10 seconds" do
+    puts "Current url: #{page.current_url}"
+    break if page.current_url.match(/backup$/)
+  end
+end
+
+Given(/^button for "([^"]*)" is available and enabled$/) do |button_text|
+  @backup_button = find_button(button_text)
+  expect(@backup_button.disabled?).to be(false)
+end
+
+Given(/^the "([^"]*)" button is disabled$/) do |button_text|
+  button = find_button(button_text, disabled: true)
+end
+
+When(/^I click the backup button$/) do
+  @backup_button.click
+  wait_for "Waiting for 'Next' button to get enabled", max: "10 seconds", sleep: "1 second" do
```

Comment:
yes, but they are usually double of the size/time which I experience on our cloud hw, these tests should be passing also on less performant virtual clouds.